ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage.hs

```diff
@@ -13,18 +12,9 @@ import           Test.Tasty (TestTree, testGroup)
 --
 
 tests :: FilePath -> TestTree
-tests tmpDir = testGroup "Storage" $
-    -- The FS tests fail for darwin on CI, see #352. So disable them for now.
-    [ FS.tests tmpDir | not darwin ] <>
+tests _tmpDir = testGroup "Storage"
```

Comment:
`_tmpdir` is unused, so we should probably remove it

Reply:
Right, yagni :)